dmutils/timing.py

```diff
@@ -0,0 +1,91 @@
+from contextlib import contextmanager
+import logging
+import sys
+from time import perf_counter, process_time
+
+from flask import request
+from flask.ctx import has_request_context
+
+
+#
+# logged_duration's defaults are exposed here so that a caller is able to defer to the default or simply use a wrapper
+# for the default when customizing the call.
+#
```

Comment:
Oh and this is something I'd like opinions on - I'm tempted here to, rather than exposing these defaults as I'm currently doing, setting them as _attributes_ on the function itself. The advantage being that to make use of them you don't have to do any additional imports - just importing the function is enough. It also has the advantage that they are more strongly bound to `logged_duration` - I can imagine here adding more tools to `timing.py` and then being in a situation where it's like "`default_message` for *what*?" But I don't really want to have to lengthen the names any more than I have to...

Thoughts?